modules/developers-guide/pages/cli-reference/dfx-canister.adoc

```diff
@@ -571,7 +573,7 @@ This command displays an error message if the request identifier is invalid or r
 
 == dfx canister set-controller
 
-Use the `+dfx canister set-controller+` command to specify the identity name or principal to use as the new **controller** for a specified canister on the Internet Computer network.
+Use the `+dfx canister set-controller+` command to specify the identity name or principal to use as the new **controller** for a specified canister on the {platform}.
 A controller identity has special rights to manage the canister it controls.
 For example, only a controlling identity can be used to install, upgrade, or delete the canister under its control.
 
```

Comment:
L632: deployed on the {IC} running locally => deployed on the local canister execution environment

Reply:
L654: local network => local canister execution environment